src/go/rpk/pkg/cli/transform/deploy.go

```diff
@@ -154,6 +170,7 @@ The --var flag can be repeated to specify multiple variables like so:
 	cmd.Flags().StringVar(&fc.functionName, "name", "", "The name of the transform")
 	cmd.Flags().Var(&fc.env, "var", "Specify an environment variable in the form of KEY=VALUE")
 	cmd.Flags().StringVar(&fc.compression, "compression", "", "Output batch compression type")
+	cmd.Flags().StringVar(&fc.fromOffset, "from-offset", "", "Process an input topic partition from this offset")
```

Comment:
Can we add the accepted values or a `; check help text for more information`?

Reply:
Absolutely. I'd drift towards the latter since it's sort of because it's (unfortunately) sort of convoluted.